packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -7658,7 +7672,9 @@ namespace Tpetra {
     {
       typename t_GlobalOrdinal_1D::HostMirror k_gblInds1D_host_this  = Kokkos::create_mirror_view(this->k_gblInds1D_);
       typename t_GlobalOrdinal_1D::HostMirror k_gblInds1D_host_graph = Kokkos::create_mirror_view(graph.k_gblInds1D_);
+      // DEEP_COPY REVIEW - NOT TESTED
```

Comment:
I'm not sure what condition would exercise this branch.
This routine is used only for comparisons in swap test.